Source/PlasticSourceControl/Private/PlasticSourceControlShell.cpp

```diff
@@ -368,6 +383,20 @@ void Terminate()
 	_ExitBackgroundCommandLineShell();
 }
 
+void SetShellIsWarmedUp()
+{
+	FScopeLock Lock(&ShellCriticalSection);
+
+	_SetShellIsWarmedUp();
+}
+
+bool GetShellIsWarmedUp()
+{
+	FScopeLock Lock(&ShellCriticalSection);
+	
```

Comment:
there is an empty line that cpplint complains about ;)

Reply:
I didn't realise about this, which made me aware of my need to change the settings of my Visual Studio to show blank spaces and newline characters 😄 Thanks!

Fixed in https://github.com/PlasticSCM/UEPlasticPlugin/pull/96/commits/2a82bf4448a59a4b2e2e3ba5c3c7a63f79360229